website/package.json

```diff
@@ -13,31 +13,34 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@edgeandnode/common": "^5.32.1",
-    "@edgeandnode/gds": "^3.8.0",
-    "@edgeandnode/go": "^4.9.0",
-    "@emotion/react": "^11.11.3",
+    "@edgeandnode/common": "^6.0.2",
+    "@edgeandnode/gds": "^5.0.0",
+    "@edgeandnode/go": "^6.0.0",
+    "@emotion/react": "^11.11.4",
     "@graphprotocol/nextra-theme": "workspace:*",
     "mixpanel-browser": "^2.49.0",
-    "next": "^14.1.0",
-    "next-seo": "^6.4.0",
+    "next": "^14.1.1",
+    "next-seo": "^6.5.0",
     "next-sitemap": "^4.2.3",
     "nextra": "^2.13.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-ga4": "^2.1.0",
-    "theme-ui": "^0.16.1",
+    "theme-ui": "^0.16.2",
     "unist-util-visit": "^5.0.0"
   },
   "devDependencies": {
     "@graphprotocol/client-cli": "^3.0.0",
     "@types/mdast": "^4.0.3",
-    "@types/mixpanel-browser": "^2.48.1",
-    "@types/react": "^18.2.55",
+    "@types/mixpanel-browser": "^2.49.0",
+    "@types/react": "^18.2.61",
     "@types/react-dom": "^18.2.19",
-    "fast-xml-parser": "^4.3.4",
+    "autoprefixer": "^10.4.18",
+    "fast-xml-parser": "^4.3.5",
     "graphql": "^16.8.1",
-    "tsx": "^4.7.0",
+    "postcss": "^8.4.35",
+    "tailwindcss": "^3.4.1",
```

Comment:
The Tailwind CSS-related changes in this PR are from [upgrading `@edgeandnode/gds`](https://github.com/edgeandnode/components/releases/tag/%40edgeandnode%2Fgds%405.0.0).